src/components/Instructions.component.js

```diff
@@ -5,14 +5,29 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
+    var myitems = this.props.items.filter(function(item) {
+      return item.length > 2
+    })
```

Comment:
you can use the arrow function syntax to one line this:

the anonymous function can be rewritten in this syntax:
```javascript
(item) => {
  return item.length > 2
}
```

arrow functions have an implicit `return` if don't use `{ }` to evaluate an entire block of code. since your `return` statement inside the callback is very short, you should make use of the implicit `return`:

```javascript
(item) => item.length > 2
```

minor detail, but if you only have one argument, you can ditch the parentheses.

all together, you get something a lot more concise, clear, and expressive:
```javascript
const myItems = this.props.items.filter(item => item.length > 2)
```